src/main/java/com/robertx22/mine_and_slash/database/data/spells/entities/SimpleProjectileEntity.java

```diff
@@ -127,6 +129,25 @@ public SimpleProjectileEntity(EntityType<? extends Entity> type, Level worldIn)
         this.setSoundEvent(SoundEvents.EMPTY);
     }
 
+    protected void moveToImpactPosition(HitResult result) {
+
+        if (result instanceof EntityHitResult enres) {
+            // the result just contains entity position, so we must clip against the AABB ourselves
+            AABB aabb = enres.getEntity().getBoundingBox().inflate(0.3D);
```

Comment:
Why is this inflated by 0.3?

Reply:
that's the default for `ProjectileUtil.getEntityHitResult`, which is used by `AbstractArrow.findHitEntity`